packages/mobile/src/web3/saga.ts

```diff
@@ -44,12 +44,25 @@ const MNEMONIC_BIT_LENGTH = MnemonicStrength.s256_24words
 export const SYNC_TIMEOUT = 2 * 60 * 1000 // 2 minutes
 const SWITCH_TO_FORNO_TIMEOUT = 15000 // if syncing takes >15 secs, suggest switch to forno
 const WEB3_MONITOR_DELAY = 100
+const BLOCK_AGE_LIMIT = 5 * 60 // if the latest block is older than 5 minutes, the node is not synced.
```

Comment:
I agree this is an overly flexible limit. I've changed this to 30 seconds, which is plenty of margin that I can't imagine any effects of network delay or slow processing on the phone, but restrictive enough to prevent the potential for transaction not mined errors.